### DISCUSSION TITLE
source=screen

### DISCUSSION THREAD

**Discussion by tomyslavs** [ultralytics/ultralytics#1006] [Q&A]
Hello,
do you plan to add a screen source to yolov8 like in yolov5?
source="screen left top width height"

**Comment by glenn-jocher** [comment]
@tomyslavs hello! Thanks for your interest in YOLOv8. We're always considering new features that could enhance the usability and functionality of our models. While we don't currently have a 'screen' source feature in YOLOv8, it's a great suggestion, and I'll certainly bring it up with the team for future consideration. Keep an eye on our updates, and feel free to check out the [Predict](https://docs.ultralytics.com/modes/predict) mode documentation for the latest options on input sources. Your feedback is valuable to us! 🚀👀
